database/seed.js

```diff
@@ -0,0 +1,23 @@
+const mysql = require('mysql');
+
+const connection = mysql.createPool({
+  connectionLimit: 100,
+  host: '127.0.0.1',
+  port: 3306,
+  user: 'root',
+  password: '',
+  database: 'related_products'
+})

```

Comment:
This code seems to be repeated in another file.

Reply:
Yes it is, I'll create a file for creating connections, thank you.